src/js/vic-v2/containers/ConfirmationPage.jsx

```diff
@@ -36,10 +41,24 @@ class ConfirmationPage extends React.Component {
     const { form, userSignedIn } = this.props;
     // If someone refreshes this page after submitting a form and it loads
     // without an empty response object, we don't want to throw errors
+    const {
+      first: firstName = '',
+      middle: middleName = '',
+      last: lastName = '',
+      suffix = ''
+    } = form.data.veteranFullName;
```

Comment:
Setting some defaults here so I don't have to check if exists when calling `toUpperCase()` later.